src/Resources/skeleton/forgottenPassword/ForgottenPasswordController.tpl.php

```diff
@@ -0,0 +1,112 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+use <?= $user_full_class_name ?>;
+use <?= $token_full_class_name ?>;
+use <?= $request_form_full_class_name ?>;
+use <?= $resetting_form_full_class_name ?>;
+use Symfony\Bundle\FrameworkBundle\Controller\<?= $parent_class_name; ?>;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
```

Comment:
I would add `@Route("/forgotten-password")` to prefix all the routes here

Reply:
Then the first would be `@Route("/request", name="app_forgotten_password_request")`, does it sound good?

Reply:
Perfect 👍 Well, we can leave `@Route("/", name="app_forgotten_password_request")` but `/request` is even better I think